en_ewt-ud-test.conllu

```diff
@@ -12223,7 +12223,7 @@
 11	be	be	AUX	VB	VerbForm=Inf	12	cop	12:cop	_
 12	key	key	ADJ	JJ	Degree=Pos	0	root	0:root	_
 13	to	to	SCONJ	IN	_	15	case	15:case	_
-14	how	how	ADV	WRB	PronType=Int	15	advmod	15:advmod	_
+14	how	how	SCONJ	WRB	PronType=Int	15	mark	15:mark	_
```

Comment:
how + ADJ

Reply:
Agreed.

Reply:
I just noticed this tree has be as the subclause predicate, but that's wrong: worthy should be the subordinate predicate. Will fix this.